SDL/Triplet.h

```diff
@@ -913,7 +912,7 @@ namespace SDL {
 
             uint16_t outerOuterLowerModuleIndex = segmentsInGPU.outerLowerModuleIndices[outerSegmentIndex];
 
-            float zOut, rtOut, deltaPhiPos, deltaPhi, betaIn, circleRadius, circleCenterX, circleCenterY;
+            float zOut, rtOut, deltaPhiPos, deltaPhi, betaIn = 0, circleRadius = -999, circleCenterX, circleCenterY;
```

Comment:
(from inspecting and cleaning up some other cases), I think it's better to make the initialization later in a place where it's clear that the good value is not going to be computed. This would be around the last `return pass` that can lead to a `success == true`.
This way we'll benefit from the compiler diagnostic to tell us that the variable is not used before it's set to something useful.

Reply:
Thank you for suggestion. However, I am still a bit confused, how can the compiler diagnostics tell us the variable is not used if we initialize them and give them values anyway?

Reply:
my comment would address the use cases of these variables inside `runTripletConstraintsAndAlgo`.

Reply:
I added an initialization after the runTripletConstraintsAndAlgo in the later commit, if the "success" is not correct, initialize all the variables as -999. Does this look ok?